components/filter-components/MapFilters.vue

```diff
@@ -74,6 +74,16 @@
                                      @deactivatedTimeline="deactivateTimeline()"
                                      @scrollHeaderIntoView="scrollHeaderIntoView()">
             </timeline-range-selector>
+
+            <button class="rounded-lg w-24 text-white ml-4
```

Comment:
This button moves about when you hover over it. I don't think this should happen. Maybe better to change the background colour or something to show the hover-over change. Thanks.